deployment/kustomizations/base/deploy-wren-ui.yaml

```diff
@@ -0,0 +1,176 @@
+            - name: OPENAI_GENERATION_MODEL
+              valueFrom:
+                configMapKeyRef:
+                  name: wren-config
+                  key: OPENAI_GENERATION_MODEL
+            - name: PG_URL
+              valueFrom:
+                secretKeyRef:
+                  name: wrenai-secrets
+                  key: PG_URL
+            # telemetry
+            - name: WREN_ENGINE_PORT
+              valueFrom:
+                configMapKeyRef:
+                  name: wren-config
+                  key: WREN_ENGINE_PORT
+            - name: WREN_AI_SERVICE_VERSION
+              valueFrom:
+                configMapKeyRef:
+                  name: wren-config
+                  key: WREN_AI_SERVICE_VERSION
+            - name: WREN_UI_VERSION
+              valueFrom:
+                configMapKeyRef:
+                  name: wren-config
+                  key: WREN_UI_VERSION
+            - name: WREN_ENGINE_VERSION
+              valueFrom:
+                configMapKeyRef:
+                  name: wren-config
+                  key: WREN_ENGINE_VERSION
+            - name: USER_UUID
+              valueFrom:
+                secretKeyRef:
+                  name: wrenai-secrets
+                  key: USER_UUID
+            - name: POSTHOG_API_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: wrenai-secrets
+                  key: POSTHOG_API_KEY
+            - name: POSTHOG_HOST
+              valueFrom:
+                secretKeyRef:
+                  name: wrenai-secrets
+                  key: POSTHOG_HOST
+            - name: TELEMETRY_ENABLED
+              valueFrom: 
+                configMapKeyRef:
+                  name: wren-config
+                  key: TELEMETRY_ENABLED
+            # client side
+            - name: NEXT_PUBLIC_USER_UUID
+              valueFrom:
+                secretKeyRef:
+                  name: wrenai-secrets
+                  key: USER_UUID
+            - name: NEXT_PUBLIC_POSTHOG_API_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: wrenai-secrets
+                  key: POSTHOG_API_KEY
+            - name: NEXT_PUBLIC_POSTHOG_HOST
+              valueFrom:
+                secretKeyRef:
+                  name: wrenai-secrets
+                  key: POSTHOG_HOST
+            - name: NEXT_PUBLIC_TELEMETRY_ENABLED
+              valueFrom: 
+                configMapKeyRef:
+                  name: wren-config
+                  key: TELEMETRY_ENABLED
+            # configs
+            - name: WREN_PRODUCT_VERSION
+              valueFrom:
+                configMapKeyRef:
+                  name: wren-config
+                  key: WREN_PRODUCT_VERSION
+          ports:
+            - containerPort: 3000
+          volumeMounts:
+            - name: wren-data
+              mountPath: /app/data
+
+        #pgbouncer is a temporary solution while there is no PG_HOST env variable implemented
```

Comment:
I think we can remove pgbouncer. In this PR, maybe we could focus on using sqlite ?

(btw, we support postgres as application database starting from UI image version `0.6.0`, PR #328 merged.)

Reply:
ok